src/App.tsx

```diff
@@ -1,26 +1,45 @@
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
+import React, { useEffect, useState } from 'react';
+import { getTodos } from './api';
```

Comment:
Ensure that `getTodos` is a stable function reference. If it changes between renders, it should be included in the `useEffect` dependency array to prevent stale closures.